### PR TITLE
pango: update to 1.55.0

### PR DESCRIPTION
**Pull request by lukaso**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 15.1.1 24B91 arm64
Xcode 16.1 16B40

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: x11/pango/Portfile [hunk 1]
```diff
@@ -59,7 +54,6 @@ configure.args-append \
                         -Dfreetype=enabled \
                         -Dintrospection=enabled \
                         -Dlibthai=disabled \
-                        -Dinstall-tests=false \
```
Comment (mascguy) [inline x11/pango/Portfile:62]: Lukas, why are you removing this?

Comment (lukaso) [reply x11/pango/Portfile:62]: I can't remember for sure, but I don't think it's available as an option anymore, so I had to remove it.

**Comment by mascguy** [comment]
Lukas, this PR is blocked by upstream issue: https://gitlab.gnome.org/GNOME/glib/-/issues/3438

I tagged you on that as well. So if you have any thoughts/suggestions for a fix, please don't hesitate to chime in there!

File: x11/pango/Portfile (hunk 1 above)
Comment (mascguy) [reply x11/pango/Portfile:62]: > I can't remember for sure, but I don't think it's available as an option anymore, so I had to remove it.

It was renamed to `build-testsuite`, for consistency with `build-examples`. Just pushed a fix.

**Pull request merged**